force-app/main/default/classes/GooglePlacesTestData.cls

```diff
@@ -70,177 +70,174 @@ public with sharing class GooglePlacesTestData {
-        String result = '{' +
-        '   \"result\" : {' +
-        '       \"address_components\" : [' +
-        '           {' +
-        '               \"long_name\" : \"Ocean Avenue\",' +
-        '               \"short_name\" : \"Ocean Ave\",' +
-        '               \"types\" : [ \"route\" ]' +
-        '           },' +
-        '           {' +
-        '               \"long_name\" : \"Del Mar\",' +
-        '               \"short_name\" : \"Del Mar\",' +
-        '               \"types\" : [ \"locality\" ]' +
-        '           },' +
-        '           {' +
-        '               \"long_name\" : \"California\",' +
-        '               \"short_name\" : \"CA\",' +
-        '               \"types\" : [ \"administrative_area_level_1\" ]' +
-        '           },' +
-        '           {' +
-        '               \"long_name\" : \"United States\",' +
-        '               \"short_name\" : \"US\",' +
-        '               \"types\" : [ \"country\" ]' +
-        '           },' +
-        '           {' +
-        '               \"long_name\" : \"92014\",' +
-        '               \"short_name\" : \"92014\",' +
-        '               \"types\" : [ \"postal_code\" ]' +
-        '           }' +
-        '       ],' +
-        '       \"formatted_phone_number\" : \"1234567890\",' +
-        '       \"name\" : \"Del Mar Church\",' +
-        '       \"website\" : \"delmarchurch\"' +
-        '       },' +
-        '   \"status\" : \"OK\"' +
-        '}';
-        return result;
-    }
-
-    public static String detailsSearchValidKeyBadAddress() {
-        String result = '{' +
-        '   \"result\" : {' +
-        '       \"address_components\" : [' +
-        '           {' +
-        '               \"long_name\" : \"Del Mar\",' +
-        '               \"short_name\" : \"Del Mar\",' +
-        '               \"types\" : [ \"locality\" ]' +
-        '           },' +
-        '           {' +
-        '               \"long_name\" : \"California\",' +
-        '               \"short_name\" : \"CA\",' +
-        '               \"types\" : [ \"administrative_area_level_1\" ]' +
-        '           },' +
-        '           {' +
-        '               \"long_name\" : \"United States\",' +
-        '               \"short_name\" : \"US\",' +
-        '               \"types\" : [ \"country\" ]' +
-        '           },' +
-        '           {' +
-        '               \"long_name\" : \"92014\",' +
-        '               \"short_name\" : \"92014\",' +
-        '               \"types\" : [ \"postal_code\" ]' +
-        '           }' +
-        '       ],' +
-        '       \"formatted_phone_number\" : \"1234567890\",' +
-        '       \"name\" : \"Del Mar Church\",' +
-        '       \"website\" : \"delmarchurch\"' +
-        '       },' +
-        '   \"status\" : \"OK\"' +
-        '}';
+    public static String apiCallResponse(String callType, String testType) {
```

Comment:
@Julian88Tex  Also re-factored all the different methods into one method that returns the mock Google data